rfc/src/template.md

```diff
@@ -10,47 +10,66 @@
 
 ## Summary
 
-Short description of the feature, i.e.: the elevator pitch. What is this feature about?
+Short (1-2 sentences) description of the feature. What is this feature about?
 
 ## User Impact
 
 Why are we doing this? How will this benefit the final user?
 
  - If this is an API change, how will that impact current users?
- - For deprecation or breaking changes, how will the transition look like?
  - If this RFC is related to change in the architecture without major user impact, think about the long term
-impact for user. I.e.: what future work will this enable.
+impact for user. I.e.: what future work will this enable. Maybe reach out to other developers to ensure you  need an RFC in the first place.
 
 ## User Experience
 
-What is the scope of this RFC? Which use cases do you have in mind? Explain how users will interact with it. Also
-please include:
+This should be a description on how users will interact with the feature.
+Users should be able to read this section and understand how to use the feature.
```

Comment:
I prefer avoiding links to old documents as mentioned before. We have a template so it reflects our current preference based on what we are learning from previous RFCs.

That said, if others feel strong about adding examples, I do prefer adding them here since we can reduce the scope of the example, and keep it coherent with the template.